es/refresher-linear-algebra.md

```diff
@@ -332,10 +330,10 @@ es una matriz simétrica de n×n, para ∇2xf(x), tal que:
 
 **56. [Matrix properties, Norm, Eigenvalue/Eigenvector, Singular-value decomposition]**
 
-&#10230; [Propiedades matriciales, Norma, Eigenvalor/Eigenvector, Descomposición de valores singulares]
+&#10230; [Propiedades de matrices, Norma, Valor propio/Vector propio, Descomposición en valores singulares]
```

Comment:
Gustavo suggested capitalizing "Value" and "Decomposition". Maybe you have to do it here also.

Reply:
Gustavo suggested the change at the next line:
> (SVD, por sus siglas en inglés de _Singular-Value Decomposition_)

what I think makes sense only because it refers to the acronym.
Do you think it's really necessary in the translation? ([wiki](https://es.wikipedia.org/wiki/Descomposici%C3%B3n_en_valores_singulares))

Reply:
Ok, you are right.